sanowret-crystal.lic

```diff
@@ -11,38 +11,36 @@ class SanowretCrystal
         { name: 'run', regex: /run/i, description: 'Single run of the script', optional: true }
       ]
     ]
-    @invalid_rooms = []
     @worn_crystal = false
     args = parse_args(arg_definitions)
     settings = get_settings
-    @adjective = settings.sanowret_adjective
-    @no_use_scripts = settings.sanowret_no_use_scripts
-    check_crystal if args.run && !hiding? && !invisible?
+    @adjective = settings.sanowret_adjective || 'sanowret'
+    @no_use_scripts = settings.sanowret_no_use_scripts || []
+    @no_use_rooms = settings.sanowret_no_use_rooms || []
+    check_crystal if args.run
     passive unless args.run
   end
 
   def use_crystal
-    return if @invalid_rooms.include?(Room.current.id)
-    return if hiding? || invisible?
```

Comment:
Redundant of `check_crystal` 

https://github.com/rpherbig/dr-scripts/pull/4435/files#diff-193d0b1c6934695a17f92dc36130239cL40